client/wfprev-war/src/main/angular/src/app/components/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="dialog-container">
     <div class="title-bar">
-        {{ dialogUsage === 'confirm-cancel' ? 'Confirm Cancel' : 'Duplicate Found' }}
+      {{ dialogUsage === 'confirm-cancel' ? 'Confirm Cancel' : 
```

Comment:
Once you start hitting 3+ levels with if/ternary operators, you might want to start considering a switch

Reply:
sounds good! updated